bundles/org.openhab.binding.modbus.lambda/src/main/resources/OH-INF/config/config-descriptions.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:lambda:modbusconfig">
+
+		<parameter name="refresh" type="integer" min="1" unit="s">
```

Comment:
Please make name, label, description (and readme.md) consistent.
1. is it Poll Interval or Refresh Interval ? 
2. Is the min 1 (xml attribute) or zero (description) ?
3. Is the right default is 20 (here) or 30 (readme) ?